pandas/tests/resample/test_period_index.py

```diff
@@ -877,3 +891,60 @@ def test_sum_min_count(self):
             [3.0, np.nan], index=PeriodIndex(["2018Q1", "2018Q2"], freq="Q-DEC")
         )
         tm.assert_series_equal(result, expected)
+
+
+def test_period_index_frequency_ME_error_message():
+    freq = "2ME"
+    msg = f"Invalid frequency: {freq}"
```

Comment:
seeing as we need to update other things: in general, for tests, I'd suggest avoid as much  logic / substitution as possible. sometimes it's inevitable, but here this would simply be `msg = "Invalid frequency: 2ME"`

Reply:
Thank you, that sounds reasonable. I simplified message as you suggested.